### PR TITLE
HOCS-3287 Check your answers screen

### PR DESCRIPTION
**Pull request by agraham19**
* Create summary component with Change link

### CODE REVIEW

**Review by gary-newbolt — COMMENTED** [review]
Again just a little bit of tidy up. Cheers!

File: src/shared/common/forms/form-repository.jsx [hunk 1]
```diff
@@ -26,19 +26,25 @@ import ExpandableCheckbox from './expandable-checkbox.jsx';
...
 import HideConditionFunctions from './../../helpers/hide-condition-functions';
 import ConfirmationWithCaseRef from './confirmation-with-case-ref.jsx';
+import ReviewField from './review-field.jsx';
 
 function defaultDataAdapter(name, data, currentValue) {
     return data[name] || currentValue;
 }
 
+function summaryBoxDataAdapter(config, data) {
+    console.log(JSON.stringify(config));
```
[the remark this answers is not in the document]
Comment (agraham19) [reply src/shared/common/forms/form-repository.jsx]: I have removed this. It looks like the lint commit hooks aren't running properly on my machine.

File: src/shared/common/forms/review-field.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            <td className='govuk-table__cell'><span className={'govuk-label govuk-label--s'}>{child.props.label}</span></td>
+            <td className={'govuk-table__cell govuk-body'}
+                id={name}
+                type={type}
+                name={name}>
+                {textValue}
+            </td>
+            <td className={'govuk-table__cell govuk-body'}><a className="govuk-body govuk-link change-link" href="" onClick={(e) => {
+                e.preventDefault();
+                console.log(`Change field with direction: ${direction}`);
```
[the remark this answers is not in the document]
Comment (agraham19) [reply src/shared/common/forms/review-field.jsx]: I have removed this

File: src/shared/pages/form-enabled.jsx [hunk 3]
```diff
@@ -108,13 +108,39 @@ function withForm(Page) {
                 });
         }
 
+        switchDirection(e, direction) {
+            e.preventDefault();
+            console.log(`page: ${JSON.stringify(this.props.page)}`);
```
[the remark this answers is not in the document]
Comment (agraham19) [reply src/shared/pages/form-enabled.jsx]: I have removed this

**Review by gary-newbolt — APPROVED** [review]
Awesome thanks, all approved

**Pull request merged**